src/coreclr/jit/compiler.h

```diff
@@ -2230,6 +2230,60 @@ typedef JitHashTable<GenTree*, JitPtrKeyFuncs<GenTree>, TestLabelAndNum> NodeToT
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #endif // DEBUG
 
+//-------------------------------------------------------------------------
+// LoopFlags: flags for the loop table.
+//
+enum LoopFlags : unsigned short
+{
+    LPFLG_EMPTY = 0,
+
+    LPFLG_DO_WHILE  = 0x0001, // it's a do-while loop (i.e ENTRY is at the TOP)
```

Comment:
All the flag names are self explanatory apart from this one, would it be better to call it `LPFLG_ENTRY_AT_TOP`

Reply:
Or `LPFLG_TOP_ENTRY`. But, `while` and `do-while` are well-known loop forms. And for this change, I don't want to rename anything, to keep the change minimal.